<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1980</width>
    <height>1280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::DockOption::AllowTabbedDocks|QMainWindow::DockOption::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Homespace">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>1911</width>
      <height>1241</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="goBackBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
   <widget class="QPushButton" name="reloadBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
   <widget class="QPushButton" name="forwardBtn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1980</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTragunyth_Browser">
    <property name="title">
     <string></string>
    </property>
    <addaction name="actionHomepage"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuTragunyth_Browser"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionHomepage">
   <property name="text">
    <string>Homepage</string>
   </property>
  </action>
  <action name="actiongoBack">
   <property name="text">
    <string>goBack</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
